src/main/kotlin/com/personio/synthetics/config/BrowserTestConfig.kt

```diff
@@ -39,6 +44,31 @@ fun BrowserTest.testFrequency(frequency: Duration) = apply {
     options.tickEvery = frequency.inWholeSeconds
 }
 
+/**
+ * Sets the advanced scheduling configuration for the synthetic browser test
+ * @param timeframes Time range and days when the test should be scheduled to run
+ * - from/to time -> pass LocalTime data type values where From value is earlier than To value
+ * - days -> pass comma separated DayOfWeek data type values
+ * @param timezone Timezone where the tests should be scheduled to run
+ * Pass ZoneId data type value, e.g. 'ZoneId.of("Europe/Berlin")'
+ * Offset value is set by the browser based on the local timezone if it is not set
```

Comment:
It's not determined by the browser. 
`value` is set by as the local timezone of the machine where the test creation runs, if the value is not explicitly set`

Reply:
Fixed